src/platform/i386/boot_comp.h

```diff
@@ -1,2 +1,2 @@
-void kern_boot_comp(void);
+void kern_boot_comp(const u32_t cpu_id);
```

Comment:
We should probably `typedef cpuid_t unsigned short` and use that everywhere.

Reply:
I will use cpuid_t instead.